src/libostree/ostree-repo-libarchive.c

```diff
@@ -526,6 +528,7 @@ aic_handle_dir (OstreeRepoArchiveImportContext *ctx,
                 GCancellable       *cancellable,
                 GError            **error)
 {
+  GLNX_AUTO_PREFIX_ERROR ("ostree-tar: Failed to import directory", error);
```

Comment:
And then

```suggestion
  GLNX_AUTO_PREFIX_ERROR ("ostree-tar: Failed to handle directory", error);
```

for consistency?

Reply:
Yep thanks, I handled it!